src/book/04-guides/01-nubots/02-camera-calibration.mdx

```diff
@@ -0,0 +1,74 @@
+---
+section: Guides
+chapter: NUbots
+title: Camera Calibration
+description: Learn how to calibrate cameras.
+slug: /guides/nubots/camera-calibration
+---
+
+Camera calibration is done to determine optimal parameters for the cameras, such as focal length and the transform from one camera to the other. More on camera calibration can be found on [the vision page](/system/nubots/vision).
+
+The calibration is done in two steps. The first is creating the nbs recording and the second is using that nbs recording to find the camera parameters.
+
+1. Change `cameratest.role` to include `support::logging::DataLogging`
```

Comment:
Maybe we should just make cameratest.role have datalogging in it by default

Reply:
It could be in there but commented out like I think it is in visualmesh.role
NUbots/NUbots#279 can address this